discovery_engine_core/core/test-fixtures/articles-invalid-clean-url.json

```diff
@@ -0,0 +1,46 @@
+[
```

Comment:
regarding the test fixtures in general, since they look like real query results(?), is there any copyright on them, and if so are we allowed to commit them in our repo?

Reply:
We've already modified the fetched articles so that they mostly contain bogus content. I ended up keeping the (modified) titles and some content to be able to tell them apart, and to also have a more realistic test. I'm unsure what the legal implications of this could be. Would it be okay if I cleaned this up in a follow-up PR?

Reply:
sure. also i don't know myself if there are any legal implications, hence the question.